README.md

```diff
@@ -33,8 +29,45 @@ to UUID generation, potentially using `Math.random()`.
 
 It's well documented that
 [`Math.random()` is not cryptographically secure](https://v8.dev/blog/math-random), by instead
-exposing users to the standard library UUID module we prevent the pitfalls that go hand in hand
-with home-grown implementations.
+exposing users to the standard library UUID we prevent the pitfalls that go hand in hand with
+home-grown implementations.
```

Comment:
It might be worth it to give credit to the UUID module for doing things well here. But, another advantage is that it gives access to using the secure RNG across environments, whereas the UUID module only has this for the web and Node.js, falling back to Math.random.

Reply:
I'm fine with throwing `uuid` under the bus here. :-)  (That `Math.random()` dependency has been a [thorn in our side](https://github.com/kelektiv/node-uuid/issues/173) for years.)

Reply:
I think we should reference the `uuid` module whenever certain decisions in this proposal are motivated by the (almost) decade of real world experience from the `uuid` module.

I'm also less concerned about "giving credit" than to show that our proposals are derived from actual real world experience (with the `Math.random()` thing being the prime example).